### ISSUE TITLE
Join scripts: don't overwrite config.toml unless explicitly told to do so

### ISSUE THREAD

**Issue by ggutoski** [axelarnetwork/axelarate-community#355]
From https://github.com/axelarnetwork/axelarate-community/issues/347#issuecomment-1051127947

> it's kinda weird that the standard way to start the node always overwrites the config in place?
It feels like overwritting the config with default config should be only happening with a special parameter no?

I agree.  We need to overhaul and simplify the join scripts.

**Comment by milapsheth** [comment]
Yeah, but we need to be careful when we do change the default config to notify users in that case. A lot of them won't notice it otherwise.
